src/components/line-input/LineInput.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useRef, useState } from "react";
+import { SLineInput } from "./styles/SLineInput";
+import { SLineInputLabel } from "./styles/SLineInputLabel";
+import { SLineInputRequiredIndicator } from "./styles/SLineInputRequiredIndicator";
+import { SLineInputRequiredIndicatorWidth } from "./styles/SLineInputRequiredIndicatorWidth";
+import { SLineInputRequiredIndicatorWrapper } from "./styles/SLineInputRequiredIndicatorWrapper";
+import { SLineInputWrapper } from "./styles/SLineInputWrapper";
+
+interface ILineInput {
+  state: "default" | "success" | "error";
+  errorMessage?: string;
+  label: string;
+  placeholder: string;
+  required?: boolean;
+  value: string;
+  onChange: (e: any) => void;
+}
+
+export const LineInput: FC<ILineInput> = ({ state, errorMessage, placeholder, label, required, value, onChange }) => {
+  const [active, setActive] = useState(false);
+  const inputRef = useRef<HTMLInputElement>();
+
+  return (
+    <SLineInputWrapper
+      onClick={() => inputRef.current?.focus()}
+      value={value}
+      onChange={onChange}
```

Comment:
onChange on a div seems strange - i think onChange and value do belong to the input below @nicolasshiken